benchopt/base.py

```diff
@@ -88,19 +95,18 @@ def is_installed(cls, env_name=None, raise_on_not_installed=None):
             returns True if no import failure has been detected.
         """
         if env_name is None:
-            import importlib
-            module = importlib.import_module(cls.__module__)
-            if (hasattr(module, 'import_ctx')
-                    and module.import_ctx.failed_import):
+            if (cls._import_ctx is not None
+                    and cls._import_ctx.failed_import):
                 if raise_on_not_installed:
-                    exc_type, value, tb = module.import_ctx.import_error
+                    exc_type, value, tb = cls._import_ctx.import_error
                     raise exc_type(value).with_traceback(tb)
                 return False
             else:
                 return True
         else:
             return _run_shell_in_conda_env(
-                f"benchopt check-install {cls.benchmark} {cls.name}",
+                f"benchopt check-install {cls._module_filename} "
```

Comment:
can you document somewhere how to use this? I don't manage to guess with check-install -h